content/browser/rewards_context.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/content/public/browser/rewards_context.h"
```

Comment:
this does not belong in `content`. In fact nothing belongs in `content`. The directory shouldn't even exist under `brave-core`

Reply:
In the future, where can we find such information?

Reply:
Removed.